.gitignore

```diff
@@ -17,9 +17,10 @@
 # general
 *~
 *.log
-tmp
-dump
 .history
+.DS_Store
```

Comment:
Ah! this should work. I must have checked in ```.DS_Store``` before adding it to gitignore. I'll remove the ```app/.DS_Store``` file